app/views/subscriptions/edit.html.erb

```diff
@@ -2,8 +2,8 @@
   <section>
     <h2>Edit <span class="provider-context"><%= current_user.provider_id %></span> Subscription</h2>
 
-    <%= form_tag(subscription_path(@concept_id), method: :put, class: 'subscription-form') do %>
-      <%= render partial: 'subscription_fields' %>
+    <%= form_tag(subscription_path(@concept_id), method: :put, class: 'subscription-form edit-subscription-form') do %>
+      <%= render partial: 'subscription_fields', locals: { edit: true } %>
```

Comment:
see my other comment about the fields, maybe `edit` local var should be more descriptive